li-hadoop-plugin/src/main/resources/dependencyPattern.json

```diff
@@ -5,7 +5,7 @@
       "message": "Open source versions(org.apache.*) of artifacts are not allowed, please use linkedin artifacts(com.linkedin.*). Open source versions will be completely blocked after March 1st 2016",
```

Comment:
Also, this is very LinkedIn-specific. Is it possible for this warning only to be applied internally? People using the LiHadoopPlugin outside of LinkedIn probably want to depend on org.apache artifacts or their own company-specific artifacts.